examples/stubbing-spying__google-analytics/README.md

```diff
@@ -1,18 +1,55 @@
 # Stubbing Google Analytics
 
```

Comment:
Is it worthwhile to have a brief explanation on how Google Analytics is instantiated and how it works? 

- I had trouble myself understanding what the `window.ga` method was or even what I was to be looking for? 

- What does the code inserted in our `index.html` mean? 

- What network calls can we expect and what can we extract from it?

Reply:
Ok, added section to the README.md walking through the Google Analytics work